src/assets/LessonContent.json

```diff
@@ -0,0 +1,277 @@
+[
```

Comment:
This file looks like a duplication of `lesson.json`. What differences does this version have?